api/app/reviews/repository.py

```diff
@@ -213,6 +214,45 @@ def get_review_complete_timeseries_by_event(event_id):
                         .all())
         return timeseries
 
+    @staticmethod
+    def get_candidate_response_ids(event_id, reviewer_user_id, reviews_required):
+        candidate_responses = db.session.query(Response.id) \

```

Comment:
Nit: rather call this `candidate_response_ids` so that it's clear to the reader that it's just the IDs and not the response objects.